[2022-10-29 03:22:14,173] {processor.py:153} INFO - Started process (PID=11579) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py
[2022-10-29 03:22:14,183] {processor.py:641} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2022-10-29 03:22:14,185] {logging_mixin.py:115} INFO - [2022-10-29 03:22:14,184] {dagbag.py:507} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py
[2022-10-29 03:22:14,191] {logging_mixin.py:115} INFO - [2022-10-29 03:22:14,189] {dagbag.py:320} ERROR - Failed to import: /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py", line 26, in <module>
    from airflow.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 23, in <module>
    from airflow.hooks.subprocess import SubprocessHook
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/subprocess.py", line 30, in <module>
    class SubprocessHook(BaseHook):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/subprocess.py", line 30, in SubprocessHook
    class SubprocessHook(BaseHook):
SystemError: unknown opcode
[2022-10-29 03:22:14,203] {processor.py:653} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py
[2022-10-29 03:22:14,277] {processor.py:161} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py took 0.110 seconds
[2022-10-29 03:22:45,003] {processor.py:153} INFO - Started process (PID=11633) to work on /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py
[2022-10-29 03:22:45,005] {processor.py:641} INFO - Processing file /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py for tasks to queue
[2022-10-29 03:22:45,006] {logging_mixin.py:115} INFO - [2022-10-29 03:22:45,005] {dagbag.py:507} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py
[2022-10-29 03:22:45,017] {logging_mixin.py:115} INFO - [2022-10-29 03:22:45,015] {dagbag.py:320} ERROR - Failed to import: /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py", line 26, in <module>
    from airflow.operators.bash import BashOperator
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 23, in <module>
    from airflow.hooks.subprocess import SubprocessHook
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/subprocess.py", line 30, in <module>
    class SubprocessHook(BaseHook):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/subprocess.py", line 30, in SubprocessHook
    class SubprocessHook(BaseHook):
SystemError: unknown opcode
[2022-10-29 03:22:45,018] {processor.py:653} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py
[2022-10-29 03:22:45,073] {processor.py:161} INFO - Processing /home/airflow/.local/lib/python3.7/site-packages/airflow/example_dags/example_complex.py took 0.075 seconds
